plugins/quota/rest.go

```diff
@@ -40,6 +40,21 @@ func QuotadStart(c transaction.TxnCtx) error {
 	return err
 }
 
+//QuotadStop is the function to stop the quota daemon
+func QuotadStop(c transaction.TxnCtx) error {
+	quotadDaemon, err := NewQuotad()
+	if err != nil {
+		return err
+	}
+
+	err = daemon.Stop(quotadDaemon, true)
```

Comment:
Quotad is a Cluster level process, so it should not be stopped if any other Volume in the same node is using Quota.

    if number_of_quota_enabled_volumes_running > 0 {
        // Do not stop, Notify the running Quotad(Restart if Restart required)
    } else {
        // Stop Quotad
    }

Reply:
Done. Got the restart part (for quotad on start for second volume or stop on only one of the volume) in with this change.